DigitalLearningSolutions.Data/Services/LearningHubSsoSecurityService.cs

```diff
@@ -56,6 +61,20 @@ public bool VerifyHash(string state, string hash)
             return false;
         }
 
+        private byte[] GetSalt()

```

Comment:
Could we change the name to make it more obvious that the salt is coming from the secret key? You don't even need to mention salt really - the method is just converting the string into bytes.